packages/core/forms/src/components/forms/OIDCForm.vue

```diff
@@ -286,7 +290,7 @@ export default {
                 case 'config-session_redis_cluster_nodes': {
                   fields.push({
                     ...field,
-                    link: 'https://docs.konghq.com/hub/kong-inc/openid-connect/#host-record',
+                    link: this.openIdConnectLink,
```

Comment:
suggestion: should be https://developer.konghq.com/plugins/openid-connect/reference/#schema--config-redis-cluster-nodes